practice_app/api/views.py

```diff
@@ -1,3 +1,51 @@
 from django.shortcuts import render
+from django.http import JsonResponse
+import requests
 
-# Create your views here.
+
+def doaj_get(request):
+    DOAJ_MAX_ROW = 10
+    
+    # Parse the parameters (query and row)
+    # Check whether the paramters are given correctly
+    try:
+        params = request.get_full_path().split('?')[-1].split('&')
+        keys = [param.split('=')[0] for param in params]
+        if keys[0] != 'query' or keys[1] != 'row':
+            raise Exception
```

Comment:
I don't think we should raise exceptions in our code. What happens when it is raised while the web app is running?

Reply:
I agree with @hakanaktas0 , you my prefer returning function with a 4** status code response.

Reply:
Since except block handles all of errors, I thought it wouldn't be a problem, but I can change them.